.github/workflows/cd.yml

```diff
@@ -307,10 +307,31 @@ on:
 
       # Options for deploying PRs. Those values should come from the PR event and should not be set manually.
       branch:
-        description: Branch to publish from. Can be used to deploy PRs to dev.
+        description: Branch or tag to publish from. Can be used to deploy PRs to dev.
```

Comment:
Maybe update "Can be used to deploy PRs to dev" so it's clearer that this will now be able to be used on the rare instance need to deploy to another env?